components/module-components/create-blogs/CreateBlogs.tsx

```diff
@@ -0,0 +1,72 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+import Button from '../../reusable-components/button/Button';
+import ReusableTable from '../../reusable-components/reusable-table/ReusableTable';
+import Filter from '../SearchAndFilter/Filter';
+import Search from '../SearchAndFilter/Search';
+import style from './createBlogs.module.scss';
+
+interface BlogPost {
+  id: string;
+  title: string;
+  tags: { name: string }[];
+  author: string;
+}
+
+const CreateBlogs = () => {
+  const [data, setData] = useState<BlogPost[]>([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch('https://staging-api.learnhub.mk/blog-posts');
+        if (!response.ok) {
+          throw new Error('Network response was not ok');
+        }
+        const result = await response.json();
+        if (result && Array.isArray(result.data)) {
```

Comment:
since we know the format of the data we are getting this check seems unnecessary